### DISCUSSION TITLE
New Failing Step Automatically added by github actions for my workflow?

### DISCUSSION THREAD

**Discussion by damageboy** [community/community#25594] [Actions]
<p>Hi,</p>
<p>I’ve started experiencing failures around 4 days ago on my webpage repo (<a href="https://github.com/damageboy/damageboy.github.io/actions" rel="nofollow noopener">https://github.com/damageboy/damageboy.github.io/actions</a>):</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/85f16042dc021d033d4887e65af18bb3bf6e2159.png" data-download-href="/uploads/short-url/j6UEfWggjiJYPXxrvRBGIaTIEZz.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/125730/181091938-702ab2d4-8aee-4be7-890f-d0a203fd1f62.png" alt="image.png" data-base62-sha1="j6UEfWggjiJYPXxrvRBGIaTIEZz" width="690" height="413" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/85f16042dc021d033d4887e65af18bb3bf6e2159_2_690x413.png, https://aws1.discourse-cdn.com/github/optimized/2X/8/85f16042dc021d033d4887e65af18bb3bf6e2159_2_1035x619.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/85f16042dc021d033d4887e65af18bb3bf6e2159_2_1380x826.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/85f16042dc021d033d4887e65af18bb3bf6e2159_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">1499×899 199 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I have not changed my workflow at all, and the new failing step seems to have been added by GitHub to MY workflow somehow…</p>
<p>Here is a before screenshot:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e2cbd57598377e00df0479593ec49f22ac8801a1.png" data-download-href="/uploads/short-url/wmkCslN6Y5SKb6Bx6QLBtVf9Or7.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/125730/181091940-a562d4d3-86c4-42de-a749-b60935004fdf.png" alt="image.png" data-base62-sha1="wmkCslN6Y5SKb6Bx6QLBtVf9Or7" width="690" height="292" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/e2cbd57598377e00df0479593ec49f22ac8801a1_2_690x292.png, https://aws1.discourse-cdn.com/github/original/2X/e/e2cbd57598377e00df0479593ec49f22ac8801a1.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/e/e2cbd57598377e00df0479593ec49f22ac8801a1.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e2cbd57598377e00df0479593ec49f22ac8801a1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">762×323 38.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>It’s easy to see that my workflow is the last step (“Deploy to github pages with Jekyll”) after which the workflow would complete (“Complete Job”).</p>
<p>Starting 4 days ago this is my workflow:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/2f44881e59a2909a1d1c1af99f73eece1c694014.png" data-download-href="/uploads/short-url/6K9gIscGFsFvAKvJcz5FthNTpYM.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/125730/181091941-214c10cb-6829-4f16-81f8-809627cd5344.png" alt="image.png" data-base62-sha1="6K9gIscGFsFvAKvJcz5FthNTpYM" width="690" height="220" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/2f44881e59a2909a1d1c1af99f73eece1c694014_2_690x220.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/2f44881e59a2909a1d1c1af99f73eece1c694014_2_1035x330.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/2/2f44881e59a2909a1d1c1af99f73eece1c694014_2_1380x440.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/2f44881e59a2909a1d1c1af99f73eece1c694014_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">2468×788 272 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>As you can tell there is a new “Post Checkout” phase which I have not added and am not aware of its origins.</p>
<p>It is obviously failing and preventing me from completing the workflow.</p>
<p>The failure seems to be completely internally related to github’s own infrastructure:</p>
<pre><code>##[error]error: could not lock config file .git/config: Permission denied
##[warning]An unsuccessful attempt was made using git command line to remove "http.extraheader" from the git config. Attempting to modify the git config file directly to remove the credential.
Removed matchers: 'checkout-git'
##[remove-matcher owner=checkout-git]
##[error]Access to the path '/home/runner/work/damageboy.github.io/damageboy.github.io/.git/config' is denied.
##[error]Exit code 1 returned from process: file name '/home/runner/runners/2.160.0/bin/Runner.PluginHost', arguments 'action "GitHub.Runner.Plugins.Repository.v1_1.CleanupTask, Runner.Plugins"'.
</code></pre>
<p>Googling these errors does not seem to lead me anywhere so far…</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Can you try to use v1 of checkout actions instead of master?</p>
<ul>
<li>uses: actions/checkout@v1</li>
</ul>
<p>I tested in my side, v1 didn’t add post checkout step . Please let us know your result.</p>

**Comment by damageboy** [comment]
<p>Thanks! That hit the spot!</p>
